src/main/scala/no/ndla/articleapi/controller/InternController.scala

```diff
@@ -46,5 +46,10 @@ trait InternController {
     get("/embedurls/:external_subject_id") {
       ArticleContentInformation.getExternalEmbedResources(params("external_subject_id"))
     }
+
+    get("/imagesinarticle/:article_id") {
```

Comment:
Jeg tenkte det kanskje var noe vi kom til å få brukt for senere også, og at det var like greit å ha et endepunkt for det. Nå er jo "list ut bilder i en artikkel" en ganske smal greie, så det er ikke sikkert vi trenger den. Hvis vi skulle trenge den senere er det ikke mye jobb å skrive på nytt.
Fjerner vi endepunktet stemmer jeg for å fjerne resten av koden også, og rejecte pull-requesten.